sketch_map_tool/routes.py

```diff
@@ -220,6 +220,15 @@ def download(uuid: str, type_: REQUEST_TYPES) -> Response:
     return send_file(file, mimetype, download_name=download_name)
 
 
+@app.route("/api/ping")
```

Comment:
Is `ping` the term you want to use? I often saw `health` or maybe `status` as endpoint name. The term `ping` also refers to ICMP requests and not necessarily to TCP/QUIC requests.

Reply:
`/status` is already in use for querying the status of a certain task (e.g. map generation). `/health` I like but seems to encompass more than one ping to on part of the system. On the other hand, one can add new parts to the `health` endpoint over time.

Reply:
renamed to `/health`